enterprise/internal/license/license.go

```diff
@@ -27,12 +27,14 @@ import (
 //
 // NOTE: If you change these fields, you MUST handle backward compatibility. Existing licenses that
 // were generated with the old fields must still work until all customers have added the new
-// license. Increment (encodedInfo).Version and formatVersion when you make backward-incompatbile
+// license. Increment (encodedInfo).Version and l.getVersion() when you make backward-incompatbile
```

Comment:
I believe this is the correct function name?
```suggestion
// license. Increment (encodedInfo).Version and l.version() when you make backward-incompatbile
```